plugins/micromega/coq_micromega.ml

```diff
@@ -1678,6 +1759,17 @@ let rec abstract_wrt_formula f1 f2 =
 
 exception CsdpNotFound
 
+let fail_csdp_not_found () =
+  flush stdout;
+  let s =
+    "Skipping the rest of this tactic: the complexity of the \
+     goal requires the use of an external tool called CSDP. \n\n\
+     However, the \"csdp\" binary is not present in the search path. \n\n\
+     Some OS distributions include Csdp (package \"coinor-csdp\" on Debian \
```

Comment:
```suggestion
     goal requires the use of an external tool called CSDP. \n\n\
     However, the \"csdp\" binary is not present in the search path. \n\n\
     Some OS distributions include Csdp (package \"coinor-csdp\" on Debian \
```
Consistency: CSDP or Csdp?

Reply:
Let's go for CSDP, it seems to be their choice in their README: https://github.com/coin-or/csdp#readme